docs/gulpfile.js

```diff
@@ -108,8 +108,9 @@ module.exports = function(gulp, IS_RELEASE_BUILD) {
   gulp.task('docs-js', ['docs-app', 'docs-html2js', 'demos', 'build'], function() {
     return gulp.src([
       'bower_components/angularytics/dist/angularytics.js',
+      'bower_components/hammerjs/hammer.js',
       'dist/angular-material.js',
-      'dist/docs/js/**/*.js'
+      'dist/docs/js/**/*.js',
     ])
```

Comment:
This means that the @programmist  code base has not been upated with Angular Material master.
This stuff should NOT be part of the PR.